### ISSUE TITLE
Remake typographic scale [Feature]

### ISSUE THREAD

**Issue by UXdave-alexander** [UKHO/admiralty-design-system#149]
The typographic scale is currently too restrictive and not enabled for REM sizing. This is a new one.

The scale here is proposed to enable 3 sizes at each tag with improved flexibility, responsiveness and accessibility

No other options needed

<img width="365" alt="image" src="https://github.com/UKHO/admiralty-design-system/assets/60921243/dc611652-fc9d-4b48-9ed6-7d04e93d1063">

**Comment by ecstaticrainbow** [comment]
As per meeting discussion y
Step 1:
- Add the typographic scale to the theme file 
```css
--admiralty-font-size--3: 0.688rem;
--admiralty-font-size--2: 0.75rem;
--admiralty-font-size--1: 0.875rem;
--admiralty-font-size-0: 1rem;
--admiralty-font-size-1: 1.375rem;
--admiralty-font-size-2: 1.5rem;
--admiralty-font-size-3: 1.75rem;
--admiralty-font-size-4: 2rem;
--admiralty-font-size-5: 2.25rem;
--admiralty-font-size-6: 2.75rem;
--admiarlty-font-size-7: 3.5rem;
```
Step 2: 
- Add the typographic scales to everything in the design system providing override varaibles
```css
font-size: var(--admiratly-expansion-heading-font-size, var(--admiralty-font-size-2));
```

Step 3: 
- Do the same as above for colours as we missed it out
```css
background-color: var(--admiralty-button-background-colour, var(--admiralty-colour-primary));
```

Step 4:
- Repeat Step 1 and 2 for font weights
```css
--admiralty-font-weight-normal: 300;
--admiralty-font-weight-medium: 400;
--admiralty-font-weight-bold: 700;

font-weight: var(--admiralty-expansion-heading-font-weight, var(--admiralty-font-weight-medium));
```

**Comment by ecstaticrainbow** [comment]
Okay Ignore Step 3, Step 3 should be a separate issue

**Comment by developernm** [comment]
[PR: Feat: Remake typographic scale](https://github.com/UKHO/admiralty-design-system/pull/286)

**Comment by ecstaticrainbow** [comment]
Fixed by #286
